cost-analyzer/templates/kubecost-cluster-controller-template.yaml

```diff
@@ -169,6 +170,17 @@ rules:
       - get
       - list
       - watch
+  # Used for namespace turndown 
+  # When cleaning a namespace, we need the ability to remove
+  # arbitrary resources (since we helm uninstall all releases in that NS first)
+  {{- if .Values.clusterController.namespaceTurndown.rbac.enabled }}
+  - apiGroups: ["*"]
+    resources: ["*"]
+    verbs:
+      - list
+      - get
+      - delete
+  {{- end }}
```

Comment:
Pretty intense permissions! I think I understand why.

Reply:
yeah. We tried to keep these constrained as much as possible - but there is no way we can defend against people trying to delete Custom Resources and stuff like that. 

Our defensive approach was to at least make this controlled under its own toggle in the helm vars